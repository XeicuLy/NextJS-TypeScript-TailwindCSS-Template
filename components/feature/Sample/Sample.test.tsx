import { render } from '@testing-library/react';

import { Sample } from './Sample';

describe('feature/Sample', () => {
  it('renders correctly', () => {
    const { container } = render(<Sample />);
    expect(container).toMatchSnapshot();
  });
});
