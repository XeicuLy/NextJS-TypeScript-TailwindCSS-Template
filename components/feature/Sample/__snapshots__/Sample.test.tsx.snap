// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feature/Sample renders correctly 1`] = `
<div>
  <h1>
    Sample
  </h1>
</div>
`;
